paparazzi-gradle-plugin/api/paparazzi-gradle-plugin.api

```diff
@@ -24,3 +24,219 @@ public abstract class app/cash/paparazzi/gradle/PrepareResourcesTask : org/gradl
+	protected fun getTitle ()Ljava/lang/String;
+	protected abstract fun registerTabs ()V
+	protected abstract fun renderBreadcrumbs (Lcom/android/build/gradle/internal/test/report/SimpleHtmlWriter;)V
+	protected final fun renderCompositeResults (Lcom/android/build/gradle/internal/test/report/SimpleHtmlWriter;Ljava/util/Map;Ljava/lang/String;)V
+	protected fun renderErrors (Lcom/android/build/gradle/internal/test/report/SimpleHtmlWriter;)V
+	protected fun renderFailures (Lcom/android/build/gradle/internal/test/report/SimpleHtmlWriter;)V
+	protected fun renderStandardError (Lcom/android/build/gradle/internal/test/report/SimpleHtmlWriter;)V
+	protected fun renderStandardOutput (Lcom/android/build/gradle/internal/test/report/SimpleHtmlWriter;)V
+	protected final fun renderTabs (Lcom/android/build/gradle/internal/test/report/SimpleHtmlWriter;)V
+	protected final fun setResults (Lapp/cash/paparazzi/gradle/reporting/CompositeTestResults;)V
+}
+
+public final class app/cash/paparazzi/gradle/reporting/ScreenshotDiffImage {
+	public fun <init> (Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;)V
+	public final fun component1 ()Ljava/lang/String;
+	public final fun component2 ()Ljava/lang/String;
+	public final fun component3 ()Ljava/lang/String;
+	public final fun component4 ()Ljava/lang/String;
+	public final fun component5 ()Ljava/lang/String;
+	public final fun copy (Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;)Lapp/cash/paparazzi/gradle/reporting/ScreenshotDiffImage;
+	public static synthetic fun copy$default (Lapp/cash/paparazzi/gradle/reporting/ScreenshotDiffImage;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;ILjava/lang/Object;)Lapp/cash/paparazzi/gradle/reporting/ScreenshotDiffImage;
+	public fun equals (Ljava/lang/Object;)Z
+	public final fun getLabel ()Ljava/lang/String;
+	public final fun getPath ()Ljava/lang/String;
+	public final fun getTestClass ()Ljava/lang/String;
+	public final fun getTestMethod ()Ljava/lang/String;
+	public final fun getTestPackage ()Ljava/lang/String;
+	public fun hashCode ()I
+	public fun toString ()Ljava/lang/String;
+}
+
+public abstract class app/cash/paparazzi/gradle/reporting/TabbedPageRenderer : org/gradle/reporting/ReportRenderer {
+	public fun <init> ()V
+	protected abstract fun getContentRenderer ()Lorg/gradle/reporting/ReportRenderer;
+	protected abstract fun getHeaderRenderer ()Lorg/gradle/reporting/ReportRenderer;
+	protected final fun getModel ()Ljava/lang/Object;
+	protected abstract fun getTitle ()Ljava/lang/String;
+	public fun render (Ljava/lang/Object;Lcom/android/build/gradle/internal/test/report/SimpleHtmlWriter;)V
+	public synthetic fun render (Ljava/lang/Object;Ljava/lang/Object;)V
+}
+
+public final class app/cash/paparazzi/gradle/reporting/TabsRenderer : org/gradle/reporting/ReportRenderer {
```

Comment:
i suspect most of this can be made internal and not part of the public plugin api